terraform/aws/security/aws-network-acl-allows-all-ports.yaml

```diff
@@ -29,7 +29,10 @@ rules:
               action = "allow"
               ...
             }
-  - metavariable-regex:
+      - metavariable-regex:
+          metavariable: $NETWORK_ACL
+          regex: ^(aws_network_acl|aws_default_network_acl)$
+    - metavariable-regex:
```

Comment:
this has broken indentation which is causing tests to fail, should be 

```yaml
metavariable-regex:
  metavariable: $PROTOCOL
  ...
```

Reply:
Fixed, thank you for pointing that out!